static/css/kiali_style.css

```diff
@@ -39,6 +39,10 @@ h2 {
   color: #FFFFFF;
 }
 
+#primary-menu ul ul li > a {
+  color: black;
+}
+
```

Comment:
This overwrite the color https://github.com/kiali/kiali.io/blob/2dafe44ce5d77fed6dd2a04cc39d535dc82f8195/static/css/kiali_style.css#L35-L40

it's necessary because we need to apply the rest of css rules